### ISSUE TITLE
Analytics on TC39 sites, e.g., the specification page

### ISSUE THREAD

**Issue by littledan** [tc39/ecma262#1167]
I don't want to invade anyone's privacy, but we may benefit from adding some basic tracking to TC39's specification. For example, if we had the download count, this would be useful to compare against the download count for specifications on Ecma's website. This data was requested by the Ecma Executive Committee in the April 2018 Face to Face meeting.

**Comment by mathiasbynens** [comment]
I agree it would be interesting to get analytics for <https://tc39.github.io/ecma262/>.

Note that for the repository itself, such data is already available at <https://github.com/tc39/ecma262/graphs/traffic>. I wouldn’t be surprised if the number of hits/visits *to the repo* already exceeds the number of PDF downloads on the Ecma’s website.

**Comment by littledan** [comment]
@mathiasbynens Thanks for the reference. Does this give us any insight on how many people go to the specification draft itself?

**Comment by mathiasbynens** [comment]
No, unfortunately. We’d need separate analytics for that.

**Comment by domenic** [comment]
FWIW we've struggled with this in the WHATWG and so far come down against tracking people when they land on specs; we have a single analytics tracker for the homepage of whatwg.org and nothing else.

I'll be curious to see what TC39 does here, and how it is received by the community. I polled once and got mixed results: https://twitter.com/domenic/status/889982507432660992

Server-side analytics seem like the best answer, but those aren't available on GitHub pages unfortunately.

**Comment by xtuc** [comment]
Note that the folks going on the spec are most likely those blocking tracker/analytics on websites. I wouldn't expect the stats to be very precise.
